Course_Part_06/src/Car.java

```diff
@@ -0,0 +1,82 @@
+public class Car {
```

Comment:
Kijk nog even goed naar het gebruik van het keyword 'this'. Wanneer is het wel en niet nodig